# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h
# Commit: 9bd2a82481f4
# Full Hash: 9bd2a82481f455019b488d0c7ce9cc2136281489
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-25 21:02:33
# Regressor Bug: 1899503
# File Overlap Count: 2
# Description:
#   Bug 1899503 - P2 PRemoteWorkerDebuggerMangaer binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebuggerManager binding.
#   
#   PRemoteWorkerDebuggerManager is the IPC that handles the registration/unregistration of remote debuggers.
# ==============================================================================

diff -r e80b18ca9165 -r 9bd2a82481f4 dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h
--- a/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h	Tue Mar 25 09:13:20 2025 +0000
+++ b/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h	Tue Mar 25 09:13:20 2025 +0000
@@ -19,6 +19,9 @@
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(RemoteWorkerDebuggerManagerParent,
                                         final)
 
+  static RefPtr<RemoteWorkerDebuggerManagerParent> CreateForProcess(
+      mozilla::ipc::Endpoint<PRemoteWorkerDebuggerManagerChild>* aChildEp);
+
   RemoteWorkerDebuggerManagerParent();
 
   mozilla::ipc::IPCResult RecvRegister(